src/writer/update-proposal.ts

```diff
@@ -49,7 +49,11 @@ export async function verify(body): Promise<any> {
 
   if (proposal.author !== body.address) return Promise.reject('Not the author');
 
-  if (space.voting?.privacy !== 'any' && msg.payload.privacy) {
+  if (
+    space.voting?.privacy !== 'any' &&
+    msg.payload.privacy &&
```

Comment:
This check will ignore empty string, and is the reason why one of the original test is failing